skins/laika/src/components/pages/membership_form/AddressType.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<fieldset>
-		<legend class="title is-size-5">{{ $t( 'membership_section_address_header_type' ) }}</legend>
```

Comment:
I named all these without `form` in there on purpose but ok..

Reply:
This way it seemed more consistent with how it was done for the donation_form component strings. I mainly wanted to add consistency here, but I see why your naming makes sense too.